### PR TITLE
app-emulation/punes: add qt6 USE-flag

### PR DESCRIPTION
**Pull request by winterheart**
Add qt6 USE-flag.

### CODE REVIEW

File: app-emulation/punes/punes-0.110.ebuild [hunk 1]
```diff
@@ -13,14 +13,20 @@ S="${WORKDIR}/puNES-${PV}"
...
+		dev-qt/qtbase:6[gui,network,opengl,widgets]
+		dev-qt/qtsvg:6
+	)
+	!qt6? (
+		dev-qt/qtcore:5
+		dev-qt/qtgui:5
+		dev-qt/qtnetwork:5
+		dev-qt/qtsvg:5
+		dev-qt/qtwidgets:5
+	)
```
Comment (ceamac) [inline app-emulation/punes/punes-0.110.ebuild:29]: USE blocks should come after the other dependencies, so both `qt6` and `!qt6` should be moved below `ffmpeg`.

File: app-emulation/punes/punes-0.110.ebuild [hunk 2]
```diff
@@ -35,13 +41,14 @@ DEPEND="
 	${RDEPEND}
 	X? ( x11-base/xorg-proto )"
 BDEPEND="
-	dev-qt/linguist-tools:5
+	qt6? ( dev-qt/qttools[linguist] )
+	!qt6? ( dev-qt/linguist-tools:5 )
```
Comment (ceamac) [inline app-emulation/punes/punes-0.110.ebuild:45]: Same here, `virtual/pkgconfig` should come first.

**Review by ceamac — COMMENTED** [review]
Please rearrange the dependencies, other than that it's good.  Thanks!

**Comment by winterheart** [comment]
> Please rearrange the dependencies, other than that it's good. Thanks!

Done.

**Pull request closed without merging**